factory.go

```diff
@@ -173,7 +173,7 @@ func (p *FilecoinParser) ParseGenesis(genesis *types.GenesisBalances, genesisTip
 			Level:       0,
 			TxTimestamp: genesisTimestamp,
 			TxTo:        balance.Key,
-			Amount:      amount.Int,
+			Amount:      amount.Int.String(),
```

Comment:
This was not really the idea. It is better to keep this as a BigInt. We want to change it on the Beryx API (convert from BigInt to String before sending the response back as JSON)